packages/terra-show-hide/src/_ShowHideButton.module.scss

```diff
@@ -0,0 +1,103 @@
+@import '~terra-mixins';
```

Comment:
What are your thoughts on prefixing all variables in this file as `--terra-show-hide-button` rather than `--terra-show-hide-`? There is a chance the actual `ShowHide.module.scss` file will need variables added in the future and those may conflict with the variables in this file

Reply:
I think this makes sense. Though I think we should also get this merged as-is. There's been a lot of requested changes on it. I can work on variable renaming before we cut the release on it

Reply:
https://github.com/cerner/terra-core/pull/1956/commits/7b97caa919fbacbb258f7d869efb994e500370ab

Reply:
@neilpfeiffer I'm planning to merge this PR as-is and then open a new PR to address the UX review comments. The OCS team can review off of that new PR.

Reply:
@bjankord +1 on merging. OCS has completed review and only really minor items were outstanding to resolve, noted them on the UX validation and created #1977 --> add anything else to the issue description from this PR I missed.